### DISCUSSION TITLE
Can I specify the number of levels and pyramid scale factor?

### DISCUSSION THREAD

**Discussion by WxpHB** [libvips/libvips#3526] [Q&A]
Hi, thanks for your great work.
I got .tif images in size of 36GB，the "vips.exe im_vips2tiff" command create  a 50GB file,and I also got .tif with almost 150GB, the command may create file in about 200GB, it's too big, . Is there any options to specify the amout of levels, such as "-pyramid-resolutions 4" ，and specify the pyramid scale factor,the default scale factor is  1 ,2,4,8,16,  can I specify the scale factor to something like "1,4,16,32,64"  or " 1,8,16 ,32"?
Thanks and regards

**Comment by jcupitt** [comment]
Hi @WxpHB,

36gb! Wow! That is very big.

Don't use `im_vips2tiff`, you need the newer vips8 interface, perhaps:

```
vips.exe copy x.v x.tif[tile,pyramid,compression=jpeg]
```

You should find it's a bit quicker.

You can shrink the image before saving, perhaps:

```
vips.exe resize x.v x.tif[tile,pyramid,compression=jpeg] 0.1
```

That'll shrink by x10 before saving, so you'll get many fewer pyramid levels.
